hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineStateManagerImpl.java

```diff
@@ -276,8 +274,7 @@ public void updatePipelineState(
           + " may have been deleted already.", pipelineID);
     } catch (IOException ex) {
       LOG.warn("Pipeline {} state update failed", pipelineID);
```

Comment:
nit: change the corresponding log level to error now that we are throwing in this case?

```suggestion
      LOG.error("Pipeline {} state update failed", pipelineID);
```